### DISCUSSION TITLE
Using 7.0.3. Able to import, but not convert, KFX-ZIP

### DISCUSSION THREAD

**Discussion by laeabbott** [apprenticeharper/DeDRM_tools#1485] [Q&A]
I import the book by selecting from the 'My Kindle Content' folder as usual. Today, however, it could not convert and place on my kindle. 

I updated my plugin from 6.8.0 to 7.0.3, but no change.

Debug file attached. Any assistance would be great!

[debugCalibreLaura.txt](https://github.com/apprenticeharper/DeDRM_tools/files/5825958/debugCalibreLaura.txt)

**Comment by ElleKayEm** [comment]
The beta DeDRM can't decrypt KFX format.  You'll need to disable KFX downloads or use Kindle for PC 1.17 which doesn't use the KFX format.  If you have an E-ink Kindle (not a Fire), you can get the older format from the website by choosing "download & transfer via usb".  Add your Kindle's serial number to DeDRM's customization to decrypt that copy.  See methods 1-3 at this post:  https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by laeabbott** [reply]
Hi ElleKay, 

Thanks. I've thankfully done all of that a while back; I've got the KFX input plugin installed and everything. Only change I made recently was to update Calibre.

Okay. A quick look at my Kindle software version shows that it's managed to update this December without asking me-- the autoupdate box is still unchecked. Not sure how it updated but I guess I will uninstall and give it another go. 

Thanks for the help!

**Reply by ElleKayEm** [reply]
You can install an older version without uninstalling the new version first.  Otherwise, some people have reported having trouble getting logged into the older version.  If you re-install a later version then K4PC 1.17, be sure to disable KFX downloads again.  See the info at the end of the post I linked which talks about replacing the updates folder to prevent updates.

**Comment by laeabbott** [comment]
Thanks for the reply! Was able to get it all working again. Seems there's
been a bigger push to sneakily install updates.

Thanks for your help!
